Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -446,6 +446,13 @@ define([
     var scratchCartesian = new Cartesian3();
     var scratchCartesianTwo = new Cartesian3();
     var scratchCartesianThree = new Cartesian3();
+    var scratchZoomViewOptions = {
+      orientation: {
```

Comment:
I don't think so. There's a branch block in `Camera.setView()` that calls out to `setView3D`, `setViewCV`, and `setView2D` with a HeadingPitchRoll instance, but those functions are private to `Camera`. I did not see any API functions in the [deployed documentation](https://cesiumjs.org/refdoc.html) for `Camera` that takes a HeadingPitchRoll as an argument.

Were you looking for a larger change to make that possible?